src/Uno.Extensions.Templates/content/unoapp-extensions/.template.config/ide.host.json

```diff
@@ -119,16 +151,73 @@
       "isVisible":true
     },
     {
-      "id": "tests",
+      "id": "unittest",
+      "name": {
+        "text": "Unit Tests"
+      },
+      "description": {
+        "text": "Includes Unit Test Project"
+      },
+      "defaultValue": "true",
+      "isVisible": true
+    },
+    {
+      "id": "uitest",
+      "name": {
+        "text": "UI Tests"
+      },
+      "description": {
+        "text": "Includes UI Test Project"
+      },
+      "defaultValue": "true",
+      "isVisible": true
+    },
+    {
+      "id": "configuration",
+      "name": {
+        "text": "Extensions - Configuration"
+      },
+      "description":
+      {
+        "text": "Uses Configuration extensions to support appsettings.json"
+      },
+      "isVisible": false
+    },
+    {
+      "id": "localization",
       "name": {
-        "text": "Testing - Unit and UI Tests"
+        "text": "Extensions - Localization"
       },
       "description":
       {
-        "text": "Includes projects for unit and UI tests"
+        "text": "Uses Localization Extensions"
       },
       "defaultValue": "true",
-      "isVisible":true
+      "isVisible": false
+    },
+    {
+      "id": "logging",
+      "name": {
+        "text": "Extensions - Logging"
+      },
+      "description":
+      {
+        "text": "Enables app logging"
+      },
+      "defaultValue": "Enabled",
+      "isVisible": false
+    },
+    {
+      "id": "markup",
+      "name": {
+        "text": "Code Style - Use C# Markup"
+      },
+      "description":
+      {
+        "text": "Uses Uno C# Markup for your UI instead of XAML"
+      },
+      "defaultValue": "Serilog",
```

Comment:
Are we missing a property? The default value here is Serilog which isnt' correct, and I don't see a seriog option